.gitignore

```diff
@@ -83,3 +83,9 @@ tmp/
 # IDEs
 .idea/
 .vscode/
+.DS_Store
+.DS_Store
+.DS_Store
+.DS_Store
+.DS_Store
+.DS_Store
```

Comment:
Why have you added the same file 6 times?
Also, why have you added this under "IDEs"?